### PR TITLE
910286: Add theme keys for interaction - .NET MAUI Chat

### PR DESCRIPTION
**Pull request by RavikumarAkuthotaSf4250**
Add theme keys for interaction - .NET MAUI Chat

### CODE REVIEW

File: MAUI/Chat/styles.md [hunk 1]
```diff
@@ -595,6 +614,14 @@ You can style the elements of a date picker message by setting values to the in-
 <td> SfChatDatePickerStroke </td>
 <td> Border color of the date picker. </td>
 </tr>
+<tr>
+<td> SfChatDateTimeInputMessageIconColor </td>
+<td> Icon background color for datetime input messsge. </td>
```
Comment (Lakshminatarajan) [inline MAUI/Chat/styles.md:619]: description is wrong. modify

File: MAUI/Chat/styles.md [hunk 2]
```diff
@@ -1145,6 +1184,22 @@ You can style the attachment button by setting values to the in-built keys of th
...
+<tr>
+<td> SfChatAttachmentBackground </td>
+<td> Background color of the attachment button. </td>
+</tr>
+<tr>
+<td> SfChatHoveredAttachmentBackground </td>
+<td> Hovered background color of the attachment button. </td>
+</tr>
+<tr>
+<td> SfChatHoveredAttachmentButtonColor </td>
```
Comment (Lakshminatarajan) [inline MAUI/Chat/styles.md:1196]: move this property to the last

File: MAUI/Chat/styles.md [hunk 3]
```diff
@@ -1178,4 +1233,31 @@ You can style the attachment button by setting values to the in-built keys of th
...
+<tr>
+<td> SfChatLoadMoreBackground </td>
+<td> Background color of the load more button. </td>
+</tr>
+<tr>
+<td> SfChatLoadMoreTextColor </td>
+<td> Text color of the load more button. </td>
+</tr>
+<tr>
+<td> SfChatLoadMoreIndicatorColor </td>
```
Comment (Lakshminatarajan) [inline MAUI/Chat/styles.md:1256]: Move this property to the last

File: MAUI/Themes/Keys.md [hunk 4]
```diff
@@ -2252,6 +2252,14 @@ This page lists the keys for each control and the element to which it is mapped
         <td> SfChatDatePickerBackground <br/><br/></td>
         <td> Background color of date picker.<br/><br/></td>
     </tr>
+    <tr>
+        <td> SfChatDateTimeInputMessageIconColor <br/><br/></td>
+        <td> Icon background color for datetime input messsge. <br/><br/></td>
```
Comment (Lakshminatarajan) [inline MAUI/Themes/Keys.md:2257]: description is wrong, modify.

Comment (RavikumarAkuthotaSf4250) [reply MAUI/Themes/Keys.md]: updated

File: MAUI/Chat/styles.md (hunk 3 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/Chat/styles.md]: Moved to the last

File: MAUI/Chat/styles.md (hunk 2 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/Chat/styles.md]: Moved to the last

File: MAUI/Chat/styles.md (hunk 1 above)
Comment (RavikumarAkuthotaSf4250) [reply MAUI/Chat/styles.md]: Updated

**Review by Lakshminatarajan — APPROVED** [review]
Approved

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**